static/js/biomedical/protein/data_processing_utils.ts

```diff
@@ -236,14 +236,14 @@ export function getDiseaseGeneAssoc(
                 }
                 diseaseID = n3.value;
                 for (const n4 of n3.neighbors) {
-                  if (n4.property !== "commonName") {
+                  if (n4.property !== "name") {
```

Comment:
nit for later: put all these constants terms in a separate files, since they get changed often.

Reply:
That's a great idea! Will implement in the upcoming PR!